commit 95fd095d9f17d4a918f0b060feda826e7d07146b (HEAD -> master)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2019-08-16T07:20:36-07:00 (7 weeks ago)
    
    TRITON-1872 Incorrect location echoed in releng/ls-missing-release-builds script
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/tools/releng/ls-missing-release-builds b/tools/releng/ls-missing-release-builds
index 5b2f7f5..6321109 100755
--- a/tools/releng/ls-missing-release-builds
+++ b/tools/releng/ls-missing-release-builds
@@ -113,6 +113,6 @@ echo "$targets" | while read target buildisprivate; do
     latest=$(mls $latest_mpath 2>/dev/null || true)
     if [[ -z "$latest" ]]; then
         echo "# $target, '$latest_mpath' does not exist" >&2
-        echo "./tools/launch-build ${PLAT_FLAVOR_ARG} -b $release $target"
+        echo "./tools/releng/launch-build ${PLAT_FLAVOR_ARG} -b $release $target"
     fi
 done
